frontend/src/components/Tabcordian.tsx

```diff
@@ -0,0 +1,107 @@
+import Link from "next/link"
+import { FC, useState } from "react"
+import { MdKeyboardArrowDown, MdKeyboardArrowRight } from 'react-icons/md'
+
+interface TabcordianLink {
+    title: string
+    url: string
+    description: string
+}
+
+interface TabcordianData {
+    title: string
+    description: Array<string>
+    linksTitle: string
+    links: Array<TabcordianLink>
+}
+
+export interface TabcordianProps {
+    data: Array<TabcordianData>
+}
+
+const Tabcordian: FC<TabcordianProps> = ({
+    data,
+  }: TabcordianProps) => {
+
+    const [active, setActive] = useState<number | null>(null)
```

Comment:
I would not use `null` and set the default state with number 0 `useState<number>(0)`. It would help coder when reading the code and by default the first tab item will be selected.

Reply:
See, I thought that originally too. Here's the issue: it opens a tab by default in that mode, since the first tab index is 0. Hence why I set it to null.... Again, since there's no defined functionality: i just went with what I assumed and ... also accepted this will all be rewritten in the not too distant future.